File: Models/Metrics/DestinyMetricsComponent.cs
namespace bulkybook.Models;

public class DestinyMetricsComponent
{
    public Dictionary<UInt32,DestinyMetricComponent> metrics {get;set;}
    /*  Type: uint32
Mapped to Manifest Database Definition: Destiny.Definitions.Presentation.DestinyPresentationNodeDefinition  */
    public UInt32 metricsRootNodeHash {get;set;}

}

File: Scripts/Models/DestinyMetricsComponent.ts
import { DestinyMetricComponent } from "./DestinyMetricComponent";
export interface DestinyMetricsComponent
{
    metrics: Map<number,DestinyMetricComponent>;
    /*  Type: uint32
Mapped to Manifest Database Definition: Destiny.Definitions.Presentation.DestinyPresentationNodeDefinition  */
    metricsRootNodeHash: number;
}